.github/workflows/ci-image-diff-sync.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         python-version: 3.9.9
     - uses: actions/setup-node@v2
       with:
-        node-version: 18
+        node-version: 16
```

Comment:
Even though we have node version specified, we are still getting the warning. I'm wondering if it's because we need to get it to 16 as per [suggestion](https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/)?